cloudify_rest_client/manager.py

```diff
@@ -114,6 +161,41 @@ def put_config(self, name, value, force=False):
         })
         return ConfigItem(response)
 
+    def get_manager(self, hostname):
+        """
+        Get a specific manager from the managers table
+        :param hostname: manager to get
+        """
+        if not hostname:
+            raise ValueError('Enter a valid hostname')
+        response = self.api.get('/managers/{0}'.format(hostname))
```

Comment:
I would join this line with the next line

Reply:
I think following the standard with the rest of the file will keep it readable.